charts/connect/README.md

```diff
@@ -83,6 +83,7 @@ helm install --set connect.applicationName=connect connect ./connect
 | connect.ingress.pathType            | string     | `"Prefix"`                         | Ingress PathType see [docs](https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types)                                                                                           |
 | connect.ingress.tls                 | list       | `[]`                               | Ingress TLS see [docs](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)                                                                                                       |
 | connect.nodeSelector                | object     | `{}`                               | [Node selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) stanza for the Connect pod                                                                  |
+| connect.priorityClassName           | string | `""` | priorityClassName to apply to the Connect API deployment resource. |
```

Comment:
Suggestion to include a link to the relevant docs here so that folks unfamiliar with the concept can learn more if they so wish.

```suggestion
| connect.priorityClassName           | string | `""` | [priorityClassName](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/) to apply to the Connect API deployment resource. |
```